elasticsearch/src/main/java/org/hibernate/search/elasticsearch/processor/impl/BatchingSharedElasticsearchWorkOrchestrator.java

```diff
@@ -49,6 +51,8 @@
 	private final BlockingQueue<Changeset> changesetQueue;
 	private final List<Changeset> changesetBuffer;
 	private final AtomicBoolean processingScheduled;
+	private boolean open;
```

Comment:
I'll add a coment for the _open_ field to point out it's guarded by the shutdownLock